test/timezones/accessors.jl

```diff
@@ -0,0 +1,29 @@
+import Base.Dates: Second
+
+warsaw = resolve("Europe/Warsaw", tzdata["europe"]...)
+fixed = FixedTimeZone("Fixed", -7200, 3600)
+
+# Determine total offset for FixedTimeZone.
+@test TimeZones.offset(fixed) == Second(-3600)
+@test_throws MethodError TimeZones.offset(warsaw)
+
+# ZonedDateTime accessors
+zdt = ZonedDateTime(DateTime(2014,6,12,23,59,58,57), fixed)
+@test TimeZones.localtime(zdt) == DateTime(2014,6,12,23,59,58,57)
+@test TimeZones.utc(zdt) == DateTime(2014,6,13,0,59,58,57)
+
+@test TimeZones.days(zdt) == 735396
+@test TimeZones.hour(zdt) == 23
```

Comment:
I'm not sure I quite understand. Wouldn't this return something other than 23? If `zdt` was created with `DateTime(2014,6,12,23,59,58,57)` + `fixed` timezone?